### ISSUE TITLE
Allow hiding specific plugins in the marketplace

### ISSUE THREAD

**Issue by Stan-vw** [piwik/piwik#21792]
## Summary

Context: Some plugins track more data than other plugins, this could be a reason for superusers to hide those plugins from the marketplace.

Solution: allow superusers to hide specific plugins from their instance' marketplace.

## Your Environment

* Matomo Version:  5.0.1, On-Premise

**Comment by sgiehl** [comment]
@Stan-vw What are your thoughts behin this request? 
As plugins can only be installed by a super user, I actually don't see much benefit in hiding specific plugins from the listings. The marketplace might be viewable for every logged in user, but as long as they can't install a plugin, there should be no risk to show all available plugins to them.

**Comment by Stan-vw** [comment]
As we're about to work on an easier checkout system where non-superusers can request a free plugin trial, it might be interesting for superusers to hide plugins that have been assessed as "permanently not going to happen on our instance" (for whichever reason). 
I don't see us picking this up anytime soon, but I thought it'd be good to put it on GH to capture the idea and allow users to express how they feel about it.

**Comment by bx80** [comment]
Unless superusers are being inundated by requests for specific plugins it seems like a lot of work for what it provides, especially if implemented per user. Good to have the idea captured anyway :+1:

**Comment by Stan-vw** [comment]
Yeah exactly, my idea originated when thinking through the "I'm a superuser and getting spammed" angle. Until that's a problem that users tell us about, I don't see this as high value.
